index.ts

```diff
@@ -74,18 +74,28 @@ if (typeof window !== 'undefined') {
   }
 }
 
+/**
+ * The severity level of a log event.
+ */
 export enum LogLevel {
   error = 0,
   warn,
   info,
   debug
 }
 
+/**
+ * Information supplied to the logger
+ * about a log event.
+ */
 export interface LogInfo {
```

Comment:
I was confused for a second looking at the Logger interface below as all the log levels had the type `LogInfo`. I thought that `LogInfo` was a level like e.g. `LogError`, `LogDebug`. Was going to suggest this is renamed to `LogData` but we already have one of those.

Some alternatives I am thinking off: `LogMessage`, `LogState`, `LogTrigger`, `LogEvent`? 

Actually could `LogInfo` and `LogData` be combined?

Reply:
Thanks, I changed it to `LogEvent`

> Actually could LogInfo and LogData be combined?

No. I think it could be error prone to allow overrides of `level` and `tag` e.g. `log.error({level: Log.info, tag: 'some-random-tag', message:....`.